setup.cfg

```diff
@@ -72,6 +72,8 @@ include_package_data = True
 # (see notes for the asterisk/`*` meaning)
 * =
   *.so
+  *.exp
```

Comment:
what's this file type?

Reply:
I think its an intermediate file produced on windows.  We probably don't need it.   I've been experimenting to fix the windows builds.

I haven't been able to get the windows build to succeed to build the c-ext.  I didn't realize it was failing after the `src` move until this PR

Reply:
Just realized windows produces `pyd` instead of `.so` so I'll fix that.

I haven't used a windows system with python in 15 years...

Reply:
Oh, right. And I think macOS has a dynlib.

FWIW, there's improvements to be made with migrating to PEP 621 and `setuptools-scm` but I don't want to prioritize this right now. Perhaps, this is something to target during the sprint...

Reply:
looks like macos is producing `.so`

_helpers_c.cpython-38-darwin.so